.github/workflows/python-test.yaml

```diff
@@ -0,0 +1,24 @@
+name: Python Tests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.12"]
```

Comment:
```suggestion
        python-version: ["3.10", "3.12"]
```

Reply:
Adding 3.10 as this is common in some environments and is the default version for workers.